pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -182,8 +182,8 @@ public class KitRequestShipping extends KitRequest implements HasDdpInstanceId {
                     + "LEFT JOIN kit_type ty ON (req.kit_type_id = ty.kit_type_id) WHERE req.ddp_participant_id = ? "
                     + "AND realm.instance_name = ?";
     public static final String SQL_SELECT_KIT =
-            "SELECT * FROM (SELECT realm.instance_name, re.dsm_kit_request_id FROM ddp_kit_request re, ddp_instance realm "
-                    + "WHERE realm.ddp_instance_id = re.ddp_instance_id) AS request "
+            "SELECT * FROM (SELECT realm.instance_name, re.dsm_kit_request_id, kt.kit_type_name FROM ddp_kit_request re, ddp_instance realm, kit_type kt "
+                    + "WHERE realm.ddp_instance_id = re.ddp_instance_id and kt.kit_type_id = re.kit_type_id) AS request "
```

Comment:
Will this new restriction (where predicate) work for all users of SQL_SELECT_KIT? Is this necessary because you added columns to the select statement?

Reply:
new column I needed to query is in another table and hence needed to join to other table. 
All the kit-requests should have a kit-type I believe.